template/studio/schemas/documents/route.js

```diff
@@ -9,41 +9,41 @@ export default {
     {
       name: 'slug',
       type: 'slug',
-      title: 'Slug',
+      title: 'Slug'
     },
     {
       name: 'page',
       type: 'reference',
       description: 'Select the page that this route should point to',
       to: [
         {
-          type: 'page',
-        },
-      ],
+          type: 'page'
+        }
+      ]
     },
     {
       name: 'includeInSitemap',
       type: 'boolean',
       title: 'Include page in sitemap',
-      description: 'For search engines. Will be added to /sitemap.xml',
+      description: 'For search engines. Will be added to /sitemap.xml'
     },
     {
       name: 'disallowRobots',
       type: 'boolean',
       title: 'Disallow in robots.txt',
-      description: 'Hide this route for search engines',
-    },
+      description: 'Hide this route for search engines'
+    }
   ],
   preview: {
     select: {
       slug: 'slug.current',
-      pageTitle: 'page.title',
+      pageTitle: 'page.title.en'
```

Comment:
Preview language could also be set from a `default: true` or the first language in the language array? This assumes that `en` is an option?

Reply:
Removed this now